### PR TITLE
add a note to fix PyTorch issue 113019

### PR DESCRIPTION
**Pull request by jingxu10**
Fixes https://github.com/pytorch/pytorch/issues/113019

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

cc @sekyondaMeta @svekars @carljparker @NicolasHug @kit1980 @subramen

### CODE REVIEW

File: prototype_source/pt2e_quant_ptq_x86_inductor.rst [hunk 1]
```diff
@@ -109,6 +109,10 @@ We will start by performing the necessary imports, capturing the FX Graph from t
 
 Next, we will have the FX Module to be quantized.
 
+.. note::
+
+   Steps below should be run inside the ``torch.no_grad()`` scope.
```
Comment (jerryzh168) [inline prototype_source/pt2e_quant_ptq_x86_inductor.rst:114]: only the lowering step I think? cc @leslie-fang-intel

Comment (leslie-fang-intel) [reply prototype_source/pt2e_quant_ptq_x86_inductor.rst:114]: Thanks for the comment @jerryzh168. Yean, to be more concrete, I believe the `lowering` and `benchmark` steps must run with ``torch.no_grad()`` scope. But since it's PTQ quant we are talking about here, I think the `prepare` and `convert` steps can also run with `torch.no_grad` context, right?

Comment (jerryzh168) [reply prototype_source/pt2e_quant_ptq_x86_inductor.rst:114]: `prepare` and `convert` can run with `torch.no_grad`, but whether these should be run in `torch.no_grad` is talking about requirement though, so the question is are these required or not?

Comment (leslie-fang-intel) [reply prototype_source/pt2e_quant_ptq_x86_inductor.rst:114]: Make sense. For PTQ, I think `prepare` and `convert` are not required to run under `torch.no_grad`. And the `lowering` and `benchmark` steps are must have. @jingxu10 could you help to make the changes accordingly?
